### PR TITLE
PodiumD release 3.0.0

### PR DESCRIPTION
**Pull request by canerortlek**
Added Keycloak externalDatabase: host: "" in values.yaml Changed tags: portaal: true in values.yaml
Upgraded the version of Open Formulieren to 2.7.9 in values.yaml Upgraded the version of Keycloak to 21.8.0 in Chart.yaml

### CODE REVIEW

File: charts/podiumd/values.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 tags:
-  portaal: false
+  portaal: true
   contact: false
```
Comment (bas-info-nl) [inline charts/podiumd/values.yaml:3]:
```suggestion
  contact: true
```

File: charts/podiumd/Chart.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ version: 1.1.21
 appVersion: "2.0.3"
```
Comment (bas-info-nl) [inline charts/podiumd/Chart.yaml:6]:
```suggestion
version: 3.0.0
appVersion: "3.0.0"
```

File: charts/podiumd/values.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 tags:
-  portaal: false
```
[the remark this answers is not in the document]
Comment (sandervdbenschede) [reply charts/podiumd/values.yaml:2]: eens met Sjoerd, de tag voor **portaal** moet juist weg en de condition bepaald de default

**Review by canerortlek — COMMENTED** [review]
Changed tag for contact from false to true in Values.yaml
Removed tag for portaal in values.yaml
Removed tag for Open Inwoner in Chart.yaml
Upgraded appVersion en version to 3.0.0 in Chart.yaml

**Review by canerortlek — COMMENTED** [review]
Added automatic functionality to check for the the latest release, select the current image version tag and making a new release including the changelogs

**Pull request merged**